lib/internal/test_runner/harness.js

```diff
@@ -124,7 +125,7 @@ function createProcessEventHandler(eventName, rootTest) {
       }
 
       rootTest.diagnostic(msg);
-      process.exitCode = kGenericUserError;
```

Comment:
should this be a breaking change?

Reply:
I don't think so. Do you have a test case in mind?

Reply:
doesn't it change the behavior of the `run` function?

Reply:
I guess it's possible when you use `run()` with `isolation: 'none'`. When you use `run()` with process isolation, the setting of the exit code still happens properly in the child processes.